templates/boilerplate-template/frontend/view-functions-with-surf/getMessageContent.ts

```diff
@@ -0,0 +1,19 @@
+import { createSurfClient } from "@thalalabs/surf";
+
+import { aptosClient } from "@/utils/aptosClient";
+import { MESSAGE_BOARD_ABI } from "@/utils/message_board_abi";
+
+export const getMessageContent = async (): Promise<string> => {
+  const content = await createSurfClient(aptosClient())
```

Comment:
I dont think we should createSurfClient on every request/component. 
We should probably create it once and use it in every component